Source/relay/Guide/Items of the Month/Garbage Tote.ash

```diff
@@ -53,28 +52,37 @@ void IOTMGarbageToteGenerateResource(ChecklistEntry [int] resource_entries)
     item_effect_description[lookupItem("broken champagne bottle")] = "Doubles +item";
     item_effect_description[lookupItem("deceased crimbo tree")] = "Absorbs damage";
 	
+    
+    if (__misc_state["Torso aware"])
+		item_charge_default[lookupItem("makeshift garbage shirt")] = 37;
+    item_charge_default[lookupItem("broken champagne bottle")] = 11;
+    item_charge_default[lookupItem("deceased crimbo tree")] = 1000;
```

Comment:
Let's group this section so that we only have to make the `Torso Awareness` check once. Something like:

```
   if (__misc_state["Torso aware"]) {
	item_charge_property[lookupItem("makeshift garbage shirt")] = "garbageShirtCharge";
        item_effect_description[lookupItem("makeshift garbage shirt")] = "Doubles statgain";
        item_charge_default[lookupItem("makeshift garbage shirt")] = 37;
    }

    item_charge_property[lookupItem("broken champagne bottle")] = "garbageChampagneCharge";
    item_effect_description[lookupItem("broken champagne bottle")] = "Doubles +item";
    item_charge_default[lookupItem("broken champagne bottle")] = 11;

    item_charge_property[lookupItem("deceased crimbo tree")] = "garbageTreeCharge";
    item_effect_description[lookupItem("deceased crimbo tree")] = "Absorbs damage";
    item_charge_default[lookupItem("deceased crimbo tree")] = 1000;
```